### ISSUE TITLE
UI broken in release 2.0.1

### ISSUE THREAD

**Issue by bmillemathias** [grafana/grafana#1829]
I've tested without problem release 2.0.0-beta1 and 2.0.0-beta3 but in release 2.0.1, the index page is totally blank with a tiny button and the text `{{alert.title}}`.
Looking a the index page, it calls resources ``/css/grafana.dark.min.1dc11bb8.css` and `/app/app.5e82909b.js` which don't exist in the debian package I've installed.

Here the resources that do exist in the package.
* ``css/grafana.dark.min.css``
* ``css/grafana.dark.min.5aa0b879.css`` (this the same duplicated file than the one above)
* ``app/app.js`` 
* ``app/app.6b375424.js`` (this the same duplicated file than the one above)

If I edit the file `/usr/share/grafana/public/views/index.html` and replace the resources with the respectively `css/grafana.dark.min.css` and `/app/app.js` it does work fine.

Perhaps the index page should only called the static named filesinstead of the dynamic named files

**Comment by torkelo** [comment]
strange, did you start the new grafana-server binary & init.d script? Sure you not accidentially started the old binary or init.d script? Cannot replicate this problem

**Comment by torkelo** [comment]
I installed the same package, app.6b375424.js is there and grafana.dark.min.5aa0b879.css is where they should. 

Did you restart grafana-server after upgrade? try sudo service grafana-server restart

**Comment by matejzero** [comment]
I had the same problem in CentOS7. After install I ran systemctl daemon-reload and systemctl start grafana-server.

What solved it for me:
systemctl stop grafana-server
rm -rf /var/lib/grafana/grafana.db
rm -rf /var/lib/grafana/sessions/*
systemctl start grafana-server

After that, it started to work. Maybe just a systemctl restart grafana-server would work?!

Matej

**Comment by torkelo** [comment]
yea, you must restart grafana-server after upgrade

**Comment by matejzero** [comment]
I did restart it. I removed the old grafana before installing a new version, but I still had to do 2 restarts to get it up. 

But then again, it might be a corner case. As I said, the 2nd restart solve my problems.

**Comment by bmillemathias** [comment]
Hello

I did restart grafana after the upgrade.
What I see on my machine 
```
ls /etc/init.d/grafana*
/etc/init.d/grafana -> /opt/grafana/current/scripts/init.sh
/etc/init.d/grafana-server
```
There are some leftover, the former should have been removed. Perhaps I did use this script to restart.

But something strange is the defaut view was not updated by the upgrade, and hence called resources that did not exist anymore.

To fix the issue, I had to perform a reinstall of version 2.0.1.
